containers/src/Data/IntMap/Internal.hs

```diff
@@ -771,10 +771,12 @@ disjoint t1@(Bin p1 m1 l1 r1) t2@(Bin p2 m2 l2 r2)
 {--------------------------------------------------------------------
   Compose
 --------------------------------------------------------------------}
--- | /O(|ab|*min(|bc|,W))/. Relate the keys of one map to the values of
+-- Relate the keys of one map to the values of
 -- the other, by using the values of the former as keys for lookups
 -- in the latter.
 --
+-- O (n * log m), where m is the size of the first argument
```

Comment:
I wonder whether this might look even nicer.

```suggestion
-- Complexity: /O(n * log m)/, where /m/ is the size of the first argument
```

Could you possibly share a screenshot of the rendered final result?

Reply:
![intmap](https://user-images.githubusercontent.com/2335822/84929191-65bbac00-b0c7-11ea-82f3-064d17befc27.png)
![map](https://user-images.githubusercontent.com/2335822/84929198-66ecd900-b0c7-11ea-9cd2-c295a39eb792.png)